drivers/SmartThings/zigbee-motion-sensor/src/test/test_ikea_motion.lua

```diff
@@ -68,62 +70,83 @@ test.register_message_test(
+                                              OnOff.ID)
+      }
+    )
+    test.socket.zigbee:__expect_send({
+      mock_device.id,
+      PowerConfiguration.attributes.BatteryVoltage:configure_reporting(mock_device,
+                                                                                    30,
+                                                                                    21600,
+                                                                                    1)
+    })
+    test.socket.zigbee:__expect_send(
+      {
+        mock_device.id,
+        zigbee_test_utils.build_mgmt_bind_request(mock_device)
+      }
+    )
+    mock_device:expect_metadata_update({ provisioning_state = "PROVISIONED" })
+  end
+)
+
+test.register_coroutine_test(
+  "OnWithTimedOff should be handled active and back to inactive after on time",
+  function()
+    local on_with_timed_off_command = OnOff.server.commands.OnWithTimedOff.build_test_rx(mock_device, 0x00, 0x0708, 0x0000)
+    local frm_ctrl = FrameCtrl(0x01)
+    on_with_timed_off_command.body.zcl_header.frame_ctrl = frm_ctrl
+    test.timer.__create_and_queue_test_time_advance_timer(0x0708/10, "oneshot")
+    test.socket.zigbee:__queue_receive(
         {
           mock_device.id,
-          zigbee_test_utils.build_mgmt_bind_request(mock_device)
+          on_with_timed_off_command,
         }
-      )
-      mock_device:expect_metadata_update({ provisioning_state = "PROVISIONED" })
-    end
+    )
+    test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.motionSensor.motion.active()))
+    test.wait_for_events()
+    test.mock_time.advance_time(180)
+    test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.motionSensor.motion.inactive()))
+  end
 )
 
 test.register_coroutine_test(
-    "OnWithTimedOff should be handled active and back to inactive after on time",
-    function()
-      local on_with_timed_off_command = OnOff.server.commands.OnWithTimedOff.build_test_rx(mock_device, 0x00, 0x0708, 0x0000)
-      local frm_ctrl = FrameCtrl(0x01)
-      on_with_timed_off_command.body.zcl_header.frame_ctrl = frm_ctrl
-      test.timer.__create_and_queue_test_time_advance_timer(0x0708/10, "oneshot")
-      test.socket.zigbee:__queue_receive(
-          {
-            mock_device.id,
-            on_with_timed_off_command,
-          }
-      )
-      test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.motionSensor.motion.active()))
-      test.wait_for_events()
-      test.mock_time.advance_time(180)
-      test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.motionSensor.motion.inactive()))
-    end
+  "ZDO Message handler and adding hub to group",
+  function()
+    local binding_table = mgmt_bind_response.BindingTableListRecord("\x6A\x9D\xC0\xFE\xFF\x5E\xCF\xD0", 0x01, 0x0006, 0x01, 0xB9F2)
```

Comment:
wanted to add a confirming test here, but not sure how to construct the packet that prompts the fallback

Reply:
Could set `binding_table_list_count` to 0x00 and not provide any entries